### ISSUE TITLE
packager repos are empty

### ISSUE THREAD

**Issue by fprilmeier** [jerakia/jerakia#138]
Hi,
wanted to give it a try as suggested by http://jerakia.io/basics/install/ ... yet rpm.packer.io gives a DNS error (same for deb.packager.io).
I managed to find https://packager.io/gh/crayfishx/jerakia but that is empty (i.e. no builds).

I see two ways to get this issue closed (other than "won't do", which always works :-) ):
* you either remove yum as an option in your installation docs and gem is the only way to install
* or you upload release files (deb and rpms) to the given locations and update your links

**Comment by beddari** [comment]
Yes, sadly it seems that the packager.io service we've been using is not being maintained any more. We're not sure which of the options we'll ultimately go for - but there is a clear ambition to have packages automatically created for e.g Debian, Ubuntu, and CentOS.

We've been thinking about perhaps starting to use the Opensuse Build Service for this but we're not yet sure. Any help towards this would be appreciated.
